jakarta.ls/src/main/resources/jax-rs.json

```diff
@@ -25,17 +25,73 @@
       }
     },
     "JAX-RS - new resource method": {
```

Comment:
Update these keys for the different methods 
Ie "JAX-RS - new GET resource method"